.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: go
 go:
-  - "1.10"
+  - "1.12"
```

Comment:
Do we have an official notion of what versions of go we support? Should this be a build matrix like we do for other languages or do we only support 1.12+?